lib/src/main/java/graphql/nadel/engine/plan/NadelExecutionPlan.kt

```diff
@@ -19,21 +20,5 @@ data class NadelExecutionPlan(
         val queryTransformTimingStep: NadelInstrumentationTimingParameters.Step,
         val resultTransformTimingStep: NadelInstrumentationTimingParameters.Step,
         val state: T,
-        val transformServiceExecutionContext: NadelTransformServiceExecutionContext?,
     )
-
-    /**
-     * Creates and returns a new [NadelExecutionPlan] that is a merging of `this` plan
-     * and the [other] plan.
-     */
-    fun merge(other: NadelExecutionPlan): NadelExecutionPlan {
```

Comment:
I don't think we ever use it and I don't think we can do it now that we store a context by transform